test/testing.cmake

```diff
@@ -427,11 +427,19 @@ if(ENABLE_METIS)
     ./msplit
     "${MDIR}/pipe.${GXT}" "pipe.smb"
     "pipe_m2_.smb"
+    2
   )
   mpi_test(msplit_3 3
     ./msplit
     "${MDIR}/pipe.${GXT}" "pipe.smb"
     "pipe_m3_.smb"
+    3
+  )
+  mpi_test(msplit_6 6
```

Comment:
It looks like this needs a 'set_test_depends(...)' call for the pipe_m2 mesh written by `msplit_2`.

Reply:
Good catch. Thanks!